.github/CODEOWNERS

```diff
@@ -1,92 +1,98 @@
 # See https://help.github.com/articles/about-codeowners/
 # for more info about CODEOWNERS file
 
+########################################################################
+#                   Updating CODEOWNERS file guide
+# - Add new entry under respective DEE team name
+# - Add new entry in alphabatical order under the team name
+# - Add common Github team owners first followed by specific ones
+# - Always add Github teams as owners instead of individual usernames
+# - Ensure that go/jds-codeowners file is up to date
+########################################################################
+
+########################################################################
+# Use go/jds-codeowners and lookup by samples folder name to find who
```

Comment:
ditto - can we switch to common go/dee-codeowners?

Reply:
See note above. I'd like to revisit this later. But changing it for the purpose of merging this PR